lib/guardian/user_guardian.rb

```diff
@@ -129,6 +129,11 @@ def can_see_profile?(user)
     true
   end
 
+  def can_see_user_actions?(user, action_types)
+    return true if @user == user || is_admin?
```

Comment:
```suggestion
    return true if @user.id == user.id || is_admin?
```

Was wondering if this might be better instead of trying to compare two `User` objects.

Reply:
That'll be better, definitely. Also, we need to check here if user is anonymous.